docs/source/command_line.rst

```diff
@@ -641,30 +690,33 @@ format into the specified directory.
     absolute filename to a list of line numbers that belong to typed
     functions in that file.
 
-``--cobertura-xml-report DIR``
-    Causes mypy to generate a Cobertura XML type checking coverage report.
+``--lineprecision-report DIR``
+    TBD
```

Comment:
Could you fill in the TBDs?

Reply:
Or alternatively, maybe punt doing this to a separate diff. These reports are barely being maintained to begin with, so I think it's ok to leave them mostly undocumented just a little while longer.

Reply:
I've added some documentation for `--lineprecision-report` and `--xml-report` and removed documentation for `--memory-xml-report`.